.eslintrc.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
       version: "27",
     },
   },
-  root: true,
+  root: '',
```

Comment:
What's the reason for this change? This property takes in a boolean value, so this does nothing but render the property invalid.